CBLClient/ValueSerializer.py

```diff
@@ -25,7 +25,7 @@ def serialize(value):
             bool_val = bool(value)
             return "true" if bool_val else "false"
         elif isinstance(value, int):
-            if value < 1000000:
+            if value < 1000000 and value > -1000000:
```

Comment:
this change require functional test testing

Reply:
I ran android functional test job with this branch. will run an ios functional job and 1 .net job with this branch too to verify.

I don't know why this is not an issue in the past, kind of related to the random function behavior change in python3.